Dockerfile

```
@@ -0,0 +1,48 @@
+FROM        ubuntu:jammy-20231004
+LABEL       Maxence Laude <maxence@laude.pro>
+
+ARG         DEBIAN_FRONTEND=noninteractive
+ARG         UEFIExtract_VERSION=A68
+
+RUN         echo 'apt::install-recommends "false";' > /etc/apt/apt.conf.d/no-install-recommends
+RUN         apt update && \
+            apt install -y git ruby ruby-dev upx-ucl p7zip-full innoextract build-essential unzip curl file && \
+            gem install bundler && \
+            apt autoremove -y && \
+            apt autoclean && \
+            apt clean && \
+            rm -rf /var/lib/apt/lists/*
+
+RUN         groupadd -g 1000 app \
+            && useradd -d /app -s /bin/bash -g app app \
+            && chown -R app:app /app
+
+RUN         git clone https://github.com/awilliam/rom-parser && \
+            cd rom-parser && \
+            make && \
+            chmod +x rom-parser && \
+            mv rom-parser /usr/local/bin
+
+WORKDIR     /app
```

Chown of /app before creation causes an error `0.102 chown: cannot access '/app': No such file or directory`
```suggestion
WORKDIR     /app

RUN         groupadd -g 1000 app \
            && useradd -d /app -s /bin/bash -g app app \
            && chown -R app:app /app

RUN         git clone https://github.com/awilliam/rom-parser && \
            cd rom-parser && \
            make && \
            chmod +x rom-parser && \
            mv rom-parser /usr/local/bin
```